src/db/tables/subscriber_email_preferences.ts

```diff
@@ -72,6 +68,38 @@ async function addEmailPreferenceForSubscriber(
   return res?.[0];
 }
 
+// TODO: modify after MNTOR-3557 - pref currently lives in two tables
+// this function only adds unsubscribe token for free reports
+async function addUnsubscribeTokenForSubscriber(
+  subscriberId: number,
+  token: string,
+) {
+  logger.info("add_unsubscribe_token_for_subscriber", {
+    subscriberId,
+  });
+
+  let res;
+  try {
+    res = await knex("subscriber_email_preferences")
+      .insert({
+        subscriber_id: subscriberId,
+        unsubscribe_token: token,
+        monthly_monitor_report_free: null,
+        monthly_monitor_report_free_at: null,
```

Comment:
we'd actually want to explicitly set these as null because there are two places where we can create a record first in the new table:
1) unsub token link
2) settings UI

We cannot predict which one happens first for a user, so we need a way to figure it out when we do the check